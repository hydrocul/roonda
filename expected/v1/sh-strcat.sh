#!/bin/sh

#################################################
# main.sh:
#################################################
cat <<\END_OF_ROONDA_SOURCE_FILE > $ROONDA_TMP_PATH/main.sh

cd $ROONDA_TMP_PATH

cat test.rd | $ROONDA_SELF_PATH --dry-run
(
    export ROONDA_TEST="Hello"
    cat test.rd | $ROONDA_SELF_PATH
)

END_OF_ROONDA_SOURCE_FILE
#################################################
# test.rd:
#################################################
cat <<\END_OF_ROONDA_SOURCE_FILE > $ROONDA_TMP_PATH/test.rd
sh v1

(echo (strcat "Hey." " " "uh... " (backticks echo aaa) " " (ref ROONDA_TEST)))
END_OF_ROONDA_SOURCE_FILE
#################################################

sh $ROONDA_TMP_PATH/main.sh
