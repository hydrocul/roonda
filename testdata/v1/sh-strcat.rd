sh v1

("sh" main.sh)

<< main.sh

cd $ROONDA_TMP_PATH

cat test.rd | $ROONDA_SELF_PATH --dry-run
(
    export ROONDA_TEST="Hello"
    cat test.rd | $ROONDA_SELF_PATH
)

main.sh >>

<< test.rd
sh v1

(echo (strcat "Hey." " " "uh... " (backticks echo aaa) " " (ref ROONDA_TEST)))
test.rd >>
