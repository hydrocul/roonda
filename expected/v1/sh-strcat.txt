#!/bin/sh

echo Hey.' ''uh... '"`echo aaa`"' '"$ROONDA_TEST"
Hey. uh... aaa Hello
